app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
I always find it confusing when adding comments. Maybe remove them and just add them if they will become relevant?

Reply:
@luktuc I read about this modules and I really would like to leave them as they ar by default